javalib/src/main/scala/java/util/Properties.scala

```diff
@@ -28,11 +32,47 @@ class Properties(protected val defaults: Properties)
   def setProperty(key: String, value: String): AnyRef =
     put(key, value)
 
-  // def load(reader: Reader): Unit
-  // def load(inStream: InputStream): Unit
-  // @deprecated("", "") def save(out: OutputStream, comments: String): Unit
-  // def store(writer: Writer, comments: String): Unit
-  // def store(out: OutputStream, comments: String): Unit
+  def load(reader: Reader): Unit =
+    loadImpl(reader)
+
+  def load(inStream: InputStream): Unit = {
+    loadImpl(new InputStreamReader(inStream, StandardCharsets.ISO_8859_1))
+  }
+
+  @deprecated("", "")
```

Comment:
this could be a little bit more specific :)

Reply:
The Javadocs just say Deprecated so not sure if that should be more specific - It looks like Java added enhanced deprecation in version 9 but I am not seeing any updates to this API.

Reply:
I looked at `java.util.Date` and it has `@Deprecated`. Maybe that is the correct one to use?

Reply:
Ok, I switched it the the Java one as other places in the `javalib` use that.